### ISSUE TITLE
Cannot read property 'completed_at' of null

### ISSUE THREAD

**Issue by jardicc** [avocode/avocode#926]
IMPORTANT: Please try to describe how did it happen. I don't have time for description.

-----
App v.2.8.0
Source UID: 17594
```
TypeError: Cannot read property 'completed_at' of null
    at t._handleAction (file:///C:/Users/bereza/AppData/Local/avocode/app-2.8.0/resources/app.asar/dist/avocode.min.js:19:8624)
    at Object.ID_13 (file:///C:/Users/bereza/AppData/Local/avocode/app-2.8.0/resources/app.asar/dist/avocode.min.js:19:3049)
    at Dispatcher._invokeCallback (C:\Users\bereza\AppData\Local\avocode\app-2.8.0\resources\app.asar\node_modules\flux\lib\Dispatcher.js:198:24)
    at Dispatcher.dispatch (C:\Users\bereza\AppData\Local\avocode\app-2.8.0\resources\app.asar\node_modules\flux\lib\Dispatcher.js:174:14)
    at e.dispatch (file:///C:/Users/bereza/AppData/Local/avocode/app-2.8.0/resources/app.asar/dist/avocode.min.js:21:8327)
    at t.e.dispatch (file:///C:/Users/bereza/AppData/Local/avocode/app-2.8.0/resources/app.asar/dist/avocode.min.js:2:20643)
    at t.assignItem (file:///C:/Users/bereza/AppData/Local/avocode/app-2.8.0/resources/app.asar/dist/avocode.min.js:15:12660)
    at e._addRemoteSlot (file:///
```

**Comment by jakubzitny** [comment]
Thanks for this issue as well @jardicc. It's a bug on all platforms, we'll be fixing it for next release.

**Comment by jakubzitny** [comment]
Hey, we've just released [Avocode 2.8.1](http://avocode.com/download/) which solves this issue. Please, check it out and let us know if you have any more problems with it.

Thanks ;)
